docs/config/metadata.rst

```diff
@@ -0,0 +1,49 @@
+Metadata
+========
+
+Workflows need metadata about the processed data to be able to process the data correctly. Currently, there are two ways how metadata can be
+communicated to Roddy workflows: (1) via the filesystem paths of files, or (2) via a metadata table.
+
+Note that this is provisional information as the plan is to streamline the respective code and define a clean interface to these and probably other
+metadata sources, such as XMLs, filesystem attributes, dedicated metadata files or databases.
+
+Filesystem-based Metadata
+-------------------------
+
+The filesystem-based approach uses of the filename-patterns with specific variables matched against existing filesystem objects.
+
+* outputBaseDirectory: The top-level directory beneath which filename-patterns for input and output files are matched or generated, respectively.
+* inputBaseDirectory: Often the same as the outputBaseDirectory, but Roddy can also have a separate input directory in which filename patterns are matched
+* outputAnalysisBaseDirectory: defaults to outputBaseDirectory
```

Comment:
What's it good for? The defAult value does not e plain it.

Reply:
1. This is work in progress.
2. You are right, but I first need to understand the code to be able to write something useful here.

Reply:
ok